modules/context/context.go

```diff
@@ -349,7 +349,7 @@ func (ctx *Context) RespHeader() http.Header {
 func (ctx *Context) SetServeHeaders(filename string) {
 	ctx.Resp.Header().Set("Content-Description", "File Transfer")
 	ctx.Resp.Header().Set("Content-Type", "application/octet-stream")
-	ctx.Resp.Header().Set("Content-Disposition", "attachment; filename="+filename)
+	ctx.Resp.Header().Set("Content-Disposition", fmt.Sprintf(`attachment; filename="%s"; filename*=UTF-8''%s`, filename, url.PathEscape(filename)))
```

Comment:
Maybe share a small helper function with `routers/common/repo.go` for this encoding?

Reply:
I merged both places 👍